auth/migrations/0001_sync_updated_at.sql

```diff
@@ -0,0 +1,10 @@
+-- Create a trigger to automatically set updated_at to current_timestamp
```

Comment:
Interesting! Will this affect all tables?

Reply:
My comment is a bit inaccurate, this creates a function to set updated_at. We can then create triggers for the various tables that want to use it to automatically update updated_at on any updates. See the subscriptions migration for where I set the trigger for that table. We can do this for any table we want.